src/runtime_src/core/tools/xbtop/xbtop.py

```diff
@@ -66,7 +67,8 @@ def running_counter(term, lock, len_x):
 # Key table
 def print_footer(term, lock, y_len):
     footer_buf = [
-        XBUtil.fg.yellow + "'n'/'p' - Next/Previous Report" + XBUtil.fg.reset,
+        XBUtil.fg.yellow + "'p'/'n' - Previous/Next Report" + XBUtil.fg.reset,
```

Comment:
Thank you for swapping the 'p' and 'n'.  It helps keep the menu and the key description consistent.